niri-config/src/lib.rs

```diff
@@ -187,6 +187,8 @@ pub struct Output {
     pub name: String,
     #[knuffel(child, unwrap(argument), default = 1.)]
     pub scale: f64,
+    #[knuffel(child, unwrap(argument), default = String::from("Normal"))]
+    pub transform: String,
```

Comment:
Make this an `enum Transform` with the same variants as `smithay::utils::Transform`, implement `FromStr` for it, and implement `From<Transform> for smithay::utils::Transform`. This way invalid values won't be accepted, and you won't have to convert manually elsewhere in the code. See `TapButtonMap` and `AccelProfile` for an example.